wdl/transforms/draft2/src/main/scala/wdl/transforms/draft2/wdlom2wom/WdlDraft2WomCommandTaskDefinitionMaker.scala

```diff
@@ -23,6 +24,9 @@ object WdlDraft2WomCommandTaskDefinitionMaker extends WomCommandTaskDefinitionMa
         OverridableInputDefinitionWithDefault(LocalName(d.unqualifiedName), d.womType, WdlWomExpression(d.expression.get, wdlTask))
     }).toList
 
+    // Figure out the start line of the workflow in the source file
+    val t: Terminal = AstTools.findTerminals(wdlTask.ast).head
```

Comment:
If you use `headOption` here and `srcLoc = t.map(SourceFileLocation(_.getLine))` below, then you can avoid errors if `AstTools.findTerminals` returns an empty list.

I'm not sure how likely that is, but since we have `Option`s we might as well use them

Reply:
Using `_` didn't work, so I added an `x` variable